posthog/permissions.py

```diff
@@ -177,6 +178,10 @@ class TeamMemberAccessPermission(BasePermission):
     message = "You don't have access to the project."
 
     def has_permission(self, request, view) -> bool:
+        if isinstance(request.successful_authenticator, ProjectSecretAPIKeyAuthentication):
+            # Ignore the team check for project secret API keys. It's handled by the ProjectSecretAPITokenPermission
+            return True
```

Comment:
Note: I still need to exclude `ProjectSecretAPIKeyAuthentication` from here because permissions are additive and this is still going to get called because this permission is registered in the `FeatureFlagViewSet`.

I considered moving `local_evaluation` and `remote_config` to a new `ViewSet`, but that's a lot more churn than I want to take on right now considering I'm also concurrently rewriting `local_evaluation` in Rust.

So hopefully this code here is temporary once we can move `local_evaluation`.

Reply:
ah yeah you're right... We should just hurry up and move this endpoint to Rust and do it there 😆

Reply:
I'm working on it!